### ISSUE TITLE
#70: repeatedly check listen/notify on pgconnection

### ISSUE THREAD

**Issue by uweschaefer** [factcast/factcast#423]
_From @uweschaefer on 2019-05-20 09:13:59 +0000 UTC_

_Copied from original issue: Mercateo/factcast#423_

**Comment by uweschaefer** [comment]
_From @codecov[bot] on 2019-05-20 09:25:46 +0000 UTC_

# [Codecov](https://codecov.io/gh/Mercateo/factcast/pull/423?src=pr&el=h1) Report
> Merging [#423](https://codecov.io/gh/Mercateo/factcast/pull/423?src=pr&el=desc) into [master](https://codecov.io/gh/Mercateo/factcast/commit/5321927ee1df2f43600eea5ef69a209156c8cc23?src=pr&el=desc) will **increase** coverage by `0.05%`.
> The diff coverage is `92.3%`.


```diff
@@            Coverage Diff             @@
##           master     #423      +/-   ##
==========================================
+ Coverage   88.87%   88.93%   +0.05%     
==========================================
  Files          78       78              
  Lines        1852     1870      +18     
  Branches      168      169       +1     
==========================================
+ Hits         1646     1663      +17     
  Misses        126      126              
- Partials       80       81       +1
```

**Comment by uweschaefer** [comment]
_From @uweschaefer on 2019-05-20 10:48:27 +0000 UTC_

closes #70
